### PR TITLE
Update varnish to be the entrypoint of the cluster

### PR DESCRIPTION
**Pull request by Zapadlo**
Tie elb-presence to the varnish instances

### CODE REVIEW

File: varnish-sidekick@.service [hunk 1]
```diff
@@ -4,21 +4,15 @@ BindsTo=varnish@%i.service
...
-ExecStart=/bin/sh -c "\
-  etcdctl set   /ft/services/varnish/healthcheck true; \
-  etcdctl mkdir /ft/services/varnish/servers; \
-  etcdctl set /ft/healthcheck/varnish-%i/path /__health; \
-  etcdctl set /ft/healthcheck/varnish-%i/categories read; \
-  ACCESS_CREDENTIALS=`/usr/bin/etcdctl get /ft/_credentials/public-access`; \
-  etcdctl set /vulcand/frontends/varnish-public-endpoint/frontend '{\"Type\":\"http\", \"BackendId\":\"vcb-varnish\", \"Route\":\"PathRegexp(`/.*`) && MethodRegexp(`GET|HEAD`) && HostRegexp(`.*-up-read\\\\.ft\\\\.com`)\"}'; \
-  etcdctl set /vulcand/frontends/varnish-public-endpoint/middlewares/auth '{\"Type\": \"sauth\", \"Middleware\": {\"Credentials\": \"'$ACCESS_CREDENTIALS'\"}}'; \
+ExecStart=/bin/sh -c '\
+  etcdctl set /vulcand/frontends/varnish-public-endpoint/frontend \'{"Type":"http", "BackendId":"vcb-varnish", "Route":"PathRegexp(`/.*`) && MethodRegexp(`GET|HEAD`) && HostRegexp(`.*-up-read\.ft\.com`)"}\'; \
```
Comment (tosan88) [inline varnish-sidekick@.service]: Do we still need this config?

Comment (Zapadlo) [inline varnish-sidekick@.service]: No idea, can someone confirm?

Comment (tosan88) [inline varnish-sidekick@.service]: If I understood correctly we want every external request to be directed to varnish, so I think we don't need this config anymore for vulcan. @Nesze can you confirm, please?

Comment (Nesze) [inline varnish-sidekick@.service]: It can be removed.

**Comment by tosan88** [comment]
:+1:

**Pull request merged**